### PR TITLE
Create best-paper-awards.md

### PR DESCRIPTION
**Pull request by RossMaciejewski**
Last year, the link to best papers was under Program -> Best Paper Awards. I'm not sure how to make that link. We aren't ready to go live yet, but help getting the link ready would be appreciated. Thanks web for all the help!

### CODE REVIEW

File: content/info/awards/best-paper-awards.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+**A Generic Framework and Library for Exploration of Small Multiples through Interactive Piling**
+<br/>
+Authors: Fritz Lekschas, Xinyi Zhou, Wei Chen, Nils Gehlenborg, Benjamin Bach, and Hanspeter Pfister
+
+
+**Data Visceralization: Enabling Deeper Understanding of Data Using Virtual Reality**
+<br/>
+Authors: Benjamin, David Brown, Bongshin Lee, Christophe Hurter, Steven Drucker, and Tim Dwyer
```
Comment (yelper) [inline content/info/awards/best-paper-awards.md:58]: Benjamin is missing a last name, perhaps?

**Comment by yelper** [comment]
Also the navigation is currently there, but commented out.  You can uncomment a whole section, or just let us know when you want it to show up on the sidebar :)

https://github.com/RossMaciejewski/ieeevis.org/blob/patch-100/_data/navigation.yml

File: content/info/awards/best-paper-awards.md (hunk 1 above)
Comment (RossMaciejewski) [reply content/info/awards/best-paper-awards.md:58]: Yes, left it for InfoVis, since I'm not sure. I sent them a note.

Comment (yelper) [reply content/info/awards/best-paper-awards.md:58]: kk we'll have some time before we link this page up, so we'll have some time :)

**Pull request merged**